doc-content/enterprise-only/project-data/preview-editor-alias-con.adoc

```diff
@@ -0,0 +1,4 @@
+[id='preview-editor-alias-con']
+= Concept of aliases
+
+In the Test Scenario (Preview) editor, once you map a header cell with a data object, the data object is removed from the *Test Editor* tab in the *Test Tools* panel. But you can still re-map a data object to another header cell by using the concept of *Alias*. By using the aliases, it is possible to specify multiple instances of the same data objects. To distinguish between them, each of them must be named differently. You simply need rename (by using an alias name) for the header cell. By doing so, the data object is again available in the *Test Editor* tab for remapping.
```

Comment:
"By using the aliases, it is possible to specify multiple instances of..."
Suggestion: "Aliases enable you to specify...."

Reply:
Changed as suggested.